src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SealedClaimPdfTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.pdfbox.pdmodel.PDDocument;
+import org.pdfbox.util.PDFTextStripper;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.tests.BaseTest;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.amount.AmountBreakDown;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SealedClaimPdfTest extends BaseTest {
+
+    private User claimant;
+
+    @Before
+    public void before() {
+        claimant = idamTestService.createCitizen();
+    }
+
+
+    @Test
+    public void shouldBeAbleToFindTestClaimDataInClaimIssueReceiptPdf() throws IOException {
+        Claim createdCase = getTestClaim();
+        String pdfAsText = textContentOf(retrievePdf("claimIssueReceipt", createdCase.getExternalId()));
+        assertionsOnClaimPdf(createdCase, pdfAsText);
+    }
+
+    @Test
+    public void shouldBeAbleToFindTestClaimDataInSealedClaimPdf() throws IOException {
+        Claim createdCase = getTestClaim();
+        String pdfAsText = textContentOf(retrievePdf("sealedClaim", createdCase.getExternalId()));
+        assertionsOnClaimPdf(createdCase, pdfAsText);
+    }
+
+    private void assertionsOnClaimPdf(Claim createdCase, String pdfAsText) {
+        assertThat(pdfAsText).contains("Claim number: " + createdCase.getReferenceNumber());
+        assertThat(pdfAsText).contains("Issued on: " + Formatting.formatDate(createdCase.getIssuedOn()));
+        assertThat(pdfAsText).contains("Name: " + createdCase.getClaimData().getClaimant().getName());
+        assertThat(pdfAsText).contains("Address: "
+            + createdCase.getClaimData().getClaimant().getAddress().getLine1() + " \n"
+            + createdCase.getClaimData().getClaimant().getAddress().getLine2() + " \n"
+            + createdCase.getClaimData().getClaimant().getAddress().getLine3() + " \n"
+            + createdCase.getClaimData().getClaimant().getAddress().getCity() + " \n"
+            + createdCase.getClaimData().getClaimant().getAddress().getPostcode());
+        assertThat(pdfAsText).contains("Name: " + createdCase.getClaimData().getDefendant().getName());
+        assertThat(pdfAsText).contains("Address: "
```

Comment:
can we please refactor out address and reuse?

Reply:
done in next commit